web/styles/popovers.css

```diff
@@ -408,6 +408,12 @@ ul {
 
         .group-name {
             font-weight: bold;
+
+            .zulip-icon {
```

Comment:
Let's use the full name of the icon `zulip-icon-triple-users`.

Reply:
done.